specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2023-09-01/Workspaces.json

```diff
@@ -423,23 +422,13 @@
         "capacityReservationLevel": {
           "type": "integer",
```

Comment:
>capacityReservationLevel [](http://example.com/codeflow?start=9&length=24)

Have you considered moving this type to a common file?

Reply:
Its not really common and unique to that resource. Also, integer enum is a bad practice for SDKs we do not wish to repeat.